scripts/docker/install-python.bash

```diff
@@ -48,4 +46,7 @@ pip install \
     oss2==2.9.0 \
     plotille==3.7 \
     seaborn==0.9.0 \
-    jsbeautifier
+    jsbeautifier \
+    yapf \
+    isort \
```

Comment:
Do we need to follow the version number of the original code?

Reply:
I had the same question. I feel that we can try sticking with the most recent version and see if their upgrading would make incompatible reformat suggestions.